docs/docs/05-configuration.md

```diff
@@ -136,6 +136,8 @@ $ snowpack dev --no-bundle
   - By default, Snowpack outputs Snowpack-related metadata such as [HMR](#hot-module-replacement) and [ENV](#environment-variables) info to a folder called `__snowpack__`. You can rename that folder with this option (e.g.: `metaDir: 'static/snowpack'`).
 - **`minify`** | `boolean` | Default: `true`
   - By default, Snowpack will minify your dependencies on `snowpack build`. Disable this by setting this config option to `minify: false`.
+- **`sourceMaps`** | `boolean` | Default: `true`
```

Comment:
New change: now the `buildOptions.sourceMaps` option only applies if `minify` is enabled. Reason is that Snowpack doesn’t really know how to map anything unless it’s the one doing the minifying. By and large, source mapping should happen in plugins.

Reply:
This is a bit confusing, I would assume that this turns source maps on/off for all file types, not just the minification use-case. If this were false, I'd assume that we ignore source maps generated by plugins. Can we do that?

Reply:
Good point. Maybe individual settings for plugins is overkill; they should just read from this one config setting.

Reply:
Resolving because now plugins use this setting. Should be more universal